R/plotreg.R

```diff
@@ -1,394 +1,574 @@
-  axis(side = 2, at = 1:num, labels = FALSE, las = 2, 
-      tck = 0, lty = 0, ...)
-  if (any(signif)) {
-    mtext(side = 2, text = labels[which(signif)], at = num + 1 - which(signif), 
-        col = signif.dark, las = 2, cex = ylab.cex, ...)
-  }
-  if (any(!signif)) {
-    mtext(side = 2, text = labels[which(!signif)], 
-        at = num + 1 - which(!signif), col = insignif.dark, las = 2, 
-        cex = ylab.cex, ...)
-  }
-  axis(side = 1, las = 0, lty = 0, ...)
-  
-  # add vertical gray lines
-  zeros <- rep(0, length(steps))
-  ends <- rep((num + 1), length(steps))
-  segments(steps, zeros, steps, ends, col = "gray", lwd = lwd.vbars)
-  segments(0, 0, 0, num + 1, lwd = lwd.zerobar, col = "grey75")
-  
-  # draw outer CIs
-  if (outer == TRUE) {
-    if (any(signif)) {
-      segments(lower.outer[signif], num + 1 - which(signif), 
-          upper.outer[signif], num + 1 - which(signif), lwd = lwd.outer, 
-          col = signif.light)
-    }
-    if (any(!signif)) {
-      segments(lower.outer[!signif], num + 1 - which(!signif), 
-          upper.outer[!signif], num + 1 - which(!signif), lwd = lwd.outer, 
-          col = insignif.light)
-    }
-  }
-  
-  # draw inner CIs
-  if (inner == TRUE) {
-    if (any(signif)) {
-      segments(lower.inner[signif], num + 1 - which(signif), 
-          upper.inner[signif], num + 1 - which(signif), lwd = lwd.inner, 
-          col = signif.medium)
+#' Create coefficient plots from statistical model output using \code{\link{ggplot2}}.
+#'
+#' Create coefficient plots of \R regression output using \code{\link{ggplot2}}.
+#'
+#'  The \code{plotreg} function produces coefficient plots (i.e., forest plots 
+#'  applied to point estimates and confidence intervals) and works much like 
+#'  the \code{\link{screenreg}}, \code{\link{texreg}}, \code{\link{htmlreg}}, 
+#'  \code{\link{matrixreg}} and \code{\link{wordreg}} functions. It accepts a 
+#'  single or multiple statistical models as input and internally extracts the
+#'  relevant data from the models. If confidence intervals are not defined 
+#'  in the extract method of a statistical model (see \link{extract} and 
+#'  \link{extract-methods}), the default standard errors are converted to 
+#'  confidence intervals. Most of the arguments work either like in the
+#'  \code{\link{screenreg}}, \code{\link{texreg}}, and \code{\link{htmlreg}} 
+#'  \code{\link{matrixreg}} and \code{\link{wordreg}} functions.It is 
+#'  possible to display the plots in two ways: using the \code{\link{facet}} 
+#'  option, one forest plot applied to point estimates and confidence intervals 
+#'  will be visualised in case there is only one model. If there is more than 
+#'  one model each one will be plotted next to the other; using the 
+#'  \code{\link{forest}} option, coefficients from one or more models will 
+#'  be grouped together and displayed as a forest plot. 
+#' 
+#'
+#' @param l A statistical model or a list of statistical models. Lists of
```

Comment:
You don't need to write out most of the parameters. Use the `@inheritParams` keyword to inherit these arguments from `matrixreg`. See the `texreg` function roxygen markup for an example.